src/components/Orders/OrderItem.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import {
+    Flex,
+    Box,
+    Image,
+    Text,
+    Heading,
+    Button,
+    VStack,
+} from '@chakra-ui/react';
+import { useHistory } from 'react-router';
+import { axiosInstance } from '../../util/config';
+import { API_BASE_URL } from '../../util/config';
+import { colors } from '../../util/constants';
+const OrderItem = ({ productId }) => {
+    const [title, setTitle] = useState('');
+    const [description, setDescription] = useState('');
+    const [images, setImages] = useState('');
```

Comment:
Since this is only 1 image, this could have been `image` and `setImage`.

Reply:
I will change it.